03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserSingleTable success!
03.02 22.06.57  INFO	ru.sfedu.crm.lab3.api.SingleTableTest:109 - updateRecordNegative:
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.SingleTableTest:19 - initRecord[0]: Start initiate record
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:User{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'}
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 206
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Client{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'inn=1234567890, status=NEW}
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 207
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Worker{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'position='Chief', employmentDate=1607126400, leaveDate=9999999999}
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 208
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.SingleTable.UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 200 not found
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.SingleTable.ClientSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 200 not found
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.SingleTable.WorkerSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 200 not found
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:92 - updateRecord[0]: try to merge(update) entity:User{id=200, name='null', birthDate=0, phoneNumber='null'}
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:94 - updateRecord[0]: entity updated!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:92 - updateRecord[0]: try to merge(update) entity:Client{id=200, name='null', birthDate=0, phoneNumber='null'inn=0, status=null}
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:94 - updateRecord[0]: entity updated!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:92 - updateRecord[0]: try to merge(update) entity:Worker{id=200, name='null', birthDate=0, phoneNumber='null'position='null', employmentDate=0, leaveDate=0}
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:94 - updateRecord[0]: entity updated!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.SingleTable.UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 200 not found
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.SingleTable.ClientSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 200 not found
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.SingleTable.WorkerSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 200 not found
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.06.57  INFO	ru.sfedu.crm.lab3.api.SingleTableTest:30 - Before run:
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserSingleTable success!
03.02 22.06.57  INFO	ru.sfedu.crm.lab3.api.SingleTableTest:74 - deleteRecordPositive:
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.SingleTableTest:19 - initRecord[0]: Start initiate record
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:User{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'}
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 212
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Client{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'inn=1234567890, status=NEW}
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 213
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Worker{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'position='Chief', employmentDate=1607126400, leaveDate=9999999999}
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 214
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:74 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab3.model.SingleTable.UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:77 - deleteRecord[1]: entity successfully deleted!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :ClientSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:74 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab3.model.SingleTable.ClientSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:77 - deleteRecord[1]: entity successfully deleted!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :WorkerSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:74 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab3.model.SingleTable.WorkerSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:77 - deleteRecord[1]: entity successfully deleted!
03.02 22.06.57  INFO	ru.sfedu.crm.lab3.api.SingleTableTest:30 - Before run:
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserSingleTable success!
03.02 22.06.57  INFO	ru.sfedu.crm.lab3.api.SingleTableTest:83 - deleteRecordNegative:
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:74 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab3.model.SingleTable.UserSingleTable
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:82 - deleteRecord[0] Record with id = 1 not found
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:83 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:74 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab3.model.SingleTable.ClientSingleTable
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:82 - deleteRecord[0] Record with id = 1 not found
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:83 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:74 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab3.model.SingleTable.WorkerSingleTable
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:82 - deleteRecord[0] Record with id = 1 not found
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:83 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.02 22.06.57  INFO	ru.sfedu.crm.lab3.api.SingleTableTest:30 - Before run:
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserSingleTable success!
03.02 22.06.57  INFO	ru.sfedu.crm.lab3.api.SingleTableTest:51 - receiveRecordByIdPositive:
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.SingleTableTest:19 - initRecord[0]: Start initiate record
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:User{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'}
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 215
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Client{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'inn=1234567890, status=NEW}
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 216
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Worker{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'position='Chief', employmentDate=1607126400, leaveDate=9999999999}
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 217
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.SingleTable.UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :ClientSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.SingleTable.ClientSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :WorkerSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.SingleTable.WorkerSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.06.57  INFO	ru.sfedu.crm.lab3.api.SingleTableTest:30 - Before run:
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserSingleTable success!
03.02 22.06.57  INFO	ru.sfedu.crm.lab3.api.SingleTableTest:63 - receiveRecordByIdNegative:
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.SingleTable.UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 1 not found
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.SingleTable.ClientSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 1 not found
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.SingleTable.WorkerSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 1 not found
03.02 22.06.57 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.06.57  INFO	ru.sfedu.crm.lab3.api.SingleTableTest:30 - Before run:
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserSingleTable success!
03.02 22.06.57  INFO	ru.sfedu.crm.lab3.api.SingleTableTest:36 - loadListPositive:
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.SingleTableTest:19 - initRecord[0]: Start initiate record
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:User{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'}
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 218
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Client{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'inn=1234567890, status=NEW}
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 219
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Worker{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'position='Chief', employmentDate=1607126400, leaveDate=9999999999}
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 220
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.06.57  INFO	ru.sfedu.crm.lab3.api.SingleTableTest:30 - Before run:
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserSingleTable success!
03.02 22.06.57  INFO	ru.sfedu.crm.lab3.api.SingleTableTest:44 - loadListNegative:
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :UserSingleTable
03.02 22.06.57 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.06.59  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:30 - Before run:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table ClientTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table WorkerTablePerClass success!
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:140 - addRecordPositive:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.TablePerClassTest:19 - initRecord[0]: Start initiate record
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:User{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 221
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Client{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'inn=1234567890, status=NEW}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 222
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Worker{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'position='Chief', employmentDate=1607126400, leaveDate=9999999999}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 223
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:30 - Before run:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table ClientTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table WorkerTablePerClass success!
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:101 - updateRecordPositive:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.TablePerClassTest:19 - initRecord[0]: Start initiate record
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:User{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 224
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Client{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'inn=1234567890, status=NEW}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 225
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Worker{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'position='Chief', employmentDate=1607126400, leaveDate=9999999999}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 226
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:92 - updateRecord[0]: try to merge(update) entity:User{id=224, name='UPDATED NAME', birthDate=817662118, phoneNumber='+78005553535'}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:94 - updateRecord[0]: entity updated!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:92 - updateRecord[0]: try to merge(update) entity:Client{id=225, name='UPDATED NAME', birthDate=817662118, phoneNumber='+78005553535'inn=1234567890, status=NEW}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:94 - updateRecord[0]: entity updated!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:92 - updateRecord[0]: try to merge(update) entity:Worker{id=226, name='UPDATED NAME', birthDate=817662118, phoneNumber='+78005553535'position='Chief', employmentDate=1607126400, leaveDate=9999999999}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:94 - updateRecord[0]: entity updated!
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:30 - Before run:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table ClientTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table WorkerTablePerClass success!
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:119 - updateRecordNegative:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.TablePerClassTest:19 - initRecord[0]: Start initiate record
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:User{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 227
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Client{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'inn=1234567890, status=NEW}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 228
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Worker{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'position='Chief', employmentDate=1607126400, leaveDate=9999999999}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 229
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.TablePerClass.UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 200 not found
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.TablePerClass.ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 200 not found
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.TablePerClass.WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 200 not found
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:92 - updateRecord[0]: try to merge(update) entity:User{id=200, name='null', birthDate=0, phoneNumber='null'}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:94 - updateRecord[0]: entity updated!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:92 - updateRecord[0]: try to merge(update) entity:Client{id=200, name='null', birthDate=0, phoneNumber='null'inn=0, status=null}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:94 - updateRecord[0]: entity updated!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:92 - updateRecord[0]: try to merge(update) entity:Worker{id=200, name='null', birthDate=0, phoneNumber='null'position='null', employmentDate=0, leaveDate=0}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:94 - updateRecord[0]: entity updated!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.TablePerClass.UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 200 not found
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.TablePerClass.ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 200 not found
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.TablePerClass.WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 200 not found
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:30 - Before run:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table ClientTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table WorkerTablePerClass success!
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:84 - deleteRecordPositive:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.TablePerClassTest:19 - initRecord[0]: Start initiate record
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:User{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 233
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Client{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'inn=1234567890, status=NEW}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 234
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Worker{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'position='Chief', employmentDate=1607126400, leaveDate=9999999999}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 235
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:74 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab3.model.TablePerClass.UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:77 - deleteRecord[1]: entity successfully deleted!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:74 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab3.model.TablePerClass.ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:77 - deleteRecord[1]: entity successfully deleted!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:74 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab3.model.TablePerClass.WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:77 - deleteRecord[1]: entity successfully deleted!
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:30 - Before run:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table ClientTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table WorkerTablePerClass success!
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:93 - deleteRecordNegative:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:74 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab3.model.TablePerClass.UserTablePerClass
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:82 - deleteRecord[0] Record with id = 1 not found
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:83 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:74 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab3.model.TablePerClass.ClientTablePerClass
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:82 - deleteRecord[0] Record with id = 1 not found
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:83 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:74 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab3.model.TablePerClass.WorkerTablePerClass
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:82 - deleteRecord[0] Record with id = 1 not found
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:83 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:30 - Before run:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table ClientTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table WorkerTablePerClass success!
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:61 - receiveRecordByIdPositive:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.TablePerClassTest:19 - initRecord[0]: Start initiate record
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:User{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 236
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Client{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'inn=1234567890, status=NEW}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 237
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Worker{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'position='Chief', employmentDate=1607126400, leaveDate=9999999999}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 238
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.TablePerClass.UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.TablePerClass.ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.TablePerClass.WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:30 - Before run:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table ClientTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table WorkerTablePerClass success!
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:73 - receiveRecordByIdNegative:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.TablePerClass.UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 1 not found
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.TablePerClass.ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 1 not found
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:58 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab3.model.TablePerClass.WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:60 - receiveRecordById[0]: Receiving complete
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:64 - receiveRecordById[0] Record with id = 1 not found
03.02 22.07.01 ERROR	ru.sfedu.crm.lab3.api.DataProviderLab3:65 - receiveRecordById[1]:java.lang.NullPointerException: null
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:30 - Before run:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table ClientTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table WorkerTablePerClass success!
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:38 - loadListPositive:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.TablePerClassTest:19 - initRecord[0]: Start initiate record
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:User{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 239
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Client{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'inn=1234567890, status=NEW}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 240
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:109 - addRecord[0]: try to save entity:Worker{id=0, name='John Smith', birthDate=817662118, phoneNumber='+78005553535'position='Chief', employmentDate=1607126400, leaveDate=9999999999}
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:112 - addRecord[1]: entity saved!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:113 - addRecord[1]: record id = 241
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:30 - Before run:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table UserTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table ClientTablePerClass success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:46 - clearTable[0]: try to clear table WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:49 - clearTable[0]: clearing table WorkerTablePerClass success!
03.02 22.07.01  INFO	ru.sfedu.crm.lab3.api.TablePerClassTest:50 - loadListNegative:
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :UserTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :ClientTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:28 - loadList[0]: try to load list of entities from table :WorkerTablePerClass
03.02 22.07.01 DEBUG	ru.sfedu.crm.lab3.api.DataProviderLab3:31 - loadList[0]: Loading success!
03.02 22.50.20  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.02 22.50.21 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserListTest
03.02 22.51.17  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.02 22.52.51  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.02 22.58.36  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.02 22.58.37 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserListTest
03.02 22.59.28  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.02 22.59.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.02 22.59.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:49 - clearTable[0]: clearing table UserList success!
03.02 22.59.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - addRecordPositive:
03.02 22.59.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.02 22.59.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.02 22.59.30 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.02 22.59.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:109 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1], phoneNumber='+78005553535'}
03.02 22.59.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - addRecord[1]: entity saved!
03.02 22.59.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:113 - addRecord[1]: record id = 242
03.02 22.59.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.02 22.59.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 20.14.55  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 20.14.57 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 20.15.32  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 20.15.34 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 20.25.59  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 20.26.01 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 20.26.01 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:49 - clearTable[0]: clearing table UserList success!
03.03 20.26.01  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - addRecordPositive:
03.03 20.26.01 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 20.26.01 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 20.26.01 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 20.26.01 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:109 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1], phoneNumber='+78005553535'}
03.03 20.26.01 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - addRecord[1]: entity saved!
03.03 20.26.01 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:113 - addRecord[1]: record id = 243
03.03 20.26.01 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 20.26.01 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 20.36.23  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 20.36.23  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - addRecordPositive:
03.03 20.36.23  INFO	ru.sfedu.crm.lab4.api.UserListTest:40 - UserList
03.03 20.41.02  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 20.41.03 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 20.49.47  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 20.49.49 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 20.51.22  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 20.51.24 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 20.51.24 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:49 - clearTable[0]: clearing table UserList success!
03.03 20.51.24  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - addRecordPositive:
03.03 20.51.24  INFO	ru.sfedu.crm.lab4.api.UserListTest:40 - UserList
03.03 20.52.36  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 20.52.38 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 20.52.38 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:49 - clearTable[0]: clearing table UserList success!
03.03 20.52.38  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - addRecordPositive:
03.03 20.52.38  INFO	ru.sfedu.crm.lab4.api.UserListTest:40 - UserList
03.03 20.52.55  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 20.52.57 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 20.52.57 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:49 - clearTable[0]: clearing table UserList success!
03.03 20.52.57  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - addRecordPositive:
03.03 20.52.57  INFO	ru.sfedu.crm.lab4.api.UserListTest:40 - UserList
03.03 20.53.19  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 20.53.22 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 20.53.22 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:49 - clearTable[0]: clearing table UserList success!
03.03 20.53.22  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - addRecordPositive:
03.03 20.53.22  INFO	ru.sfedu.crm.lab4.api.UserListTest:40 - UserList
03.03 20.53.41  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 20.53.42 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 20.53.43 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:49 - clearTable[0]: clearing table UserList success!
03.03 20.53.43  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - addRecordPositive:
03.03 20.53.43  INFO	ru.sfedu.crm.lab4.api.UserListTest:40 - UserList
03.03 20.53.43 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 20.53.43 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 20.53.43 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 20.53.43 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:109 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1], phoneNumber='+78005553535'}
03.03 20.53.43 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - addRecord[1]: entity saved!
03.03 20.53.43 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:113 - addRecord[1]: record id = 244
03.03 20.53.43 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 20.53.43 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 20.54.56  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 20.54.57 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 20.54.57 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:49 - clearTable[0]: clearing table UserList success!
03.03 20.54.57  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - addRecordPositive:
03.03 20.54.57  INFO	ru.sfedu.crm.lab4.api.UserListTest:40 - UserList
03.03 20.54.57 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 20.54.57 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 20.54.57 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 20.54.57 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:109 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1], phoneNumber='+78005553535'}
03.03 20.54.57 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - addRecord[1]: entity saved!
03.03 20.54.57 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:113 - addRecord[1]: record id = 245
03.03 20.54.57 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 20.54.57 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 20.56.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 20.56.27 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 20.57.09  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 20.57.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.07.44  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.07.45 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.10.24  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.10.24  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - addRecordPositive:
03.03 21.10.24  INFO	ru.sfedu.crm.lab4.api.UserListTest:40 - UserList
03.03 21.10.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.10.26 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.10.26 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.10.26 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:109 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.10.26 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - addRecord[1]: entity saved!
03.03 21.10.26 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:113 - addRecord[1]: record id = 246
03.03 21.10.26 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.10.26 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.10.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.10.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - addRecordPositive:
03.03 21.10.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:40 - UserList
03.03 21.10.31 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.10.31 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.11.23  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.11.23  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - addRecordPositive:
03.03 21.11.23  INFO	ru.sfedu.crm.lab4.api.UserListTest:40 - UserList
03.03 21.11.23 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.11.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:109 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.11.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - addRecord[1]: entity saved!
03.03 21.11.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:113 - addRecord[1]: record id = 247
03.03 21.11.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.11.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.11.28  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.11.28  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - addRecordPositive:
03.03 21.11.28  INFO	ru.sfedu.crm.lab4.api.UserListTest:40 - UserList
03.03 21.11.28 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.11.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:109 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.11.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - addRecord[1]: entity saved!
03.03 21.11.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:113 - addRecord[1]: record id = 248
03.03 21.11.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.11.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.15.16  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.15.16  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - loadListPositive:
03.03 21.15.16 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.15.18 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:109 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.15.18 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - addRecord[1]: entity saved!
03.03 21.15.18 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:113 - addRecord[1]: record id = 249
03.03 21.15.18 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.15.18 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.15.38  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.15.38  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - loadListPositive:
03.03 21.15.40 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.15.40 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.29.51  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.29.51  INFO	ru.sfedu.crm.lab4.api.UserListTest:46 - addRecordPositive:
03.03 21.29.51  INFO	ru.sfedu.crm.lab4.api.UserListTest:47 - UserList
03.03 21.29.51 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.29.53 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.29.53 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: entity saved!
03.03 21.29.53 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:116 - addRecord[1]: record id = 250
03.03 21.29.53 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.29.53 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.30.00  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.30.01 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.30.50  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.30.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.30.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:52 - clearTable[0]: clearing table UserList success!
03.03 21.30.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:46 - addRecordPositive:
03.03 21.30.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:47 - UserList
03.03 21.30.52 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.30.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.30.52 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:120 - addRecord[0]: saving error!
03.03 21.30.52 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:121 - addRecord[1]:javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not prepare statement
03.03 21.30.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.30.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.33.20  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.33.21 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.33.21 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:52 - clearTable[0]: clearing table UserList success!
03.03 21.33.21  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - loadListPositive:
03.03 21.33.21 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.33.21 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.33.58  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.33.59 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.33.59 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:52 - clearTable[0]: clearing table UserList success!
03.03 21.33.59  INFO	ru.sfedu.crm.lab4.api.UserListTest:46 - addRecordPositive:
03.03 21.33.59  INFO	ru.sfedu.crm.lab4.api.UserListTest:47 - UserList
03.03 21.33.59 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.33.59 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.33.59 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.33.59 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.33.59 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:120 - addRecord[0]: saving error!
03.03 21.33.59 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:121 - addRecord[1]:javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not prepare statement
03.03 21.33.59 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.33.59 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.35.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.35.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:46 - addRecordPositive:
03.03 21.35.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:47 - UserList
03.03 21.35.26 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.35.26 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.35.26 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.35.26 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.35.26 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: entity saved!
03.03 21.35.26 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:116 - addRecord[1]: record id = 253
03.03 21.35.26 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.35.26 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.36.33  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.36.35 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.36.35 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:52 - clearTable[0]: clearing table UserList success!
03.03 21.36.35  INFO	ru.sfedu.crm.lab4.api.UserListTest:46 - addRecordPositive:
03.03 21.36.35  INFO	ru.sfedu.crm.lab4.api.UserListTest:47 - UserList
03.03 21.36.35 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.36.35 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.36.35 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.36.35 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.36.35 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: entity saved!
03.03 21.36.35 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:116 - addRecord[1]: record id = 254
03.03 21.36.35 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.36.35 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.36.39  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.36.40 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.36.40 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:52 - clearTable[0]: clearing table UserList success!
03.03 21.36.40  INFO	ru.sfedu.crm.lab4.api.UserListTest:46 - addRecordPositive:
03.03 21.36.40  INFO	ru.sfedu.crm.lab4.api.UserListTest:47 - UserList
03.03 21.36.40 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.36.40 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.36.40 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.36.40 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.36.40 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: entity saved!
03.03 21.36.40 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:116 - addRecord[1]: record id = 255
03.03 21.36.40 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.36.40 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.36.57  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.36.58 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.36.58 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:52 - clearTable[0]: clearing table UserList success!
03.03 21.36.58  INFO	ru.sfedu.crm.lab4.api.UserListTest:46 - addRecordPositive:
03.03 21.36.58  INFO	ru.sfedu.crm.lab4.api.UserListTest:47 - UserList
03.03 21.36.58 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.36.58 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.36.58 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.36.58 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.36.58 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: entity saved!
03.03 21.36.58 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:116 - addRecord[1]: record id = 256
03.03 21.36.58 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.36.58 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.42.39  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.42.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.42.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.42.41  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - loadListPositive:
03.03 21.42.41 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.42.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.42.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 21.42.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 257
03.03 21.42.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.42.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.43.15  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.43.17 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.43.17 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.43.17  INFO	ru.sfedu.crm.lab4.api.UserListTest:46 - loadListNegative:
03.03 21.43.17 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.43.17 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.44.03  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.44.04 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.44.04 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.44.04  INFO	ru.sfedu.crm.lab4.api.UserListTest:54 - receiveRecordByIdPositive:
03.03 21.44.04 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.44.04 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.44.04 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 21.44.04 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 258
03.03 21.44.04 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.44.04 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.44.04 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 21.44.04 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 21.44.38  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.44.39 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.44.39 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.44.39  INFO	ru.sfedu.crm.lab4.api.UserListTest:62 - receiveRecordByIdNegative:
03.03 21.44.39 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.44.39 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.45.26  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.45.27 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.45.27 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.45.27  INFO	ru.sfedu.crm.lab4.api.UserListTest:62 - receiveRecordByIdNegative:
03.03 21.45.27 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 21.45.27 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 21.45.27 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:66 - receiveRecordById[0] Record with id = 1 not found
03.03 21.45.27 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:67 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 21.45.53  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.45.55 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.45.55 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.45.55  INFO	ru.sfedu.crm.lab4.api.UserListTest:69 - deleteRecordPositive:
03.03 21.45.55 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.45.55 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.45.55 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 21.45.55 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 259
03.03 21.45.55 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.45.55 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.45.55 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.list.UserList
03.03 21.45.55 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:79 - deleteRecord[1]: entity successfully deleted!
03.03 21.46.41  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.46.43 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.46.43 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.46.43  INFO	ru.sfedu.crm.lab4.api.UserListTest:76 - deleteRecordPositive:
03.03 21.46.43 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.46.43 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.46.43 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 21.46.43 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 260
03.03 21.46.43 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.list.UserList
03.03 21.46.43 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:84 - deleteRecord[0] Record with id = 1 not found
03.03 21.46.43 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:85 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.03 21.46.51  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.46.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.46.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.46.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:76 - deleteRecordPositive:
03.03 21.46.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.list.UserList
03.03 21.46.52 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:84 - deleteRecord[0] Record with id = 1 not found
03.03 21.46.52 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:85 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.03 21.47.29  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.47.31 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.47.31 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.47.31  INFO	ru.sfedu.crm.lab4.api.UserListTest:82 - updateRecordPositive:
03.03 21.47.31 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.47.31 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.47.31 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 21.47.31 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 261
03.03 21.47.31 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.47.31 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.47.31 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:94 - updateRecord[0]: try to merge(update) entity:UserList{id=261, name='UPDATED NAME', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.47.31 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:96 - updateRecord[0]: entity updated!
03.03 21.48.39  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.48.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.48.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.48.41  INFO	ru.sfedu.crm.lab4.api.UserListTest:91 - updateRecordNegative:
03.03 21.48.41 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.48.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.48.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 21.48.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 262
03.03 21.48.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 21.48.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 21.48.41 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:66 - receiveRecordById[0] Record with id = 200 not found
03.03 21.48.41 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:67 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 21.48.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:94 - updateRecord[0]: try to merge(update) entity:UserList{id=200, name='null', OrdersId=[], phoneNumber='null'}
03.03 21.48.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:96 - updateRecord[0]: entity updated!
03.03 21.48.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 21.48.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 21.48.41 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:66 - receiveRecordById[0] Record with id = 200 not found
03.03 21.48.41 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:67 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 21.48.51  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:102 - addRecordPositive:
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:103 - UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 264
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:82 - updateRecordPositive:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 265
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:94 - updateRecord[0]: try to merge(update) entity:UserList{id=265, name='UPDATED NAME', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:96 - updateRecord[0]: entity updated!
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:91 - updateRecordNegative:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 266
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 21.48.52 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:66 - receiveRecordById[0] Record with id = 200 not found
03.03 21.48.52 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:67 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:94 - updateRecord[0]: try to merge(update) entity:UserList{id=200, name='null', OrdersId=[], phoneNumber='null'}
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:96 - updateRecord[0]: entity updated!
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 21.48.52 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:66 - receiveRecordById[0] Record with id = 200 not found
03.03 21.48.52 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:67 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:69 - deleteRecordPositive:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 268
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.list.UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:79 - deleteRecord[1]: entity successfully deleted!
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:76 - deleteRecordNegative:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.list.UserList
03.03 21.48.52 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:84 - deleteRecord[0] Record with id = 1 not found
03.03 21.48.52 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:85 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:54 - receiveRecordByIdPositive:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 269
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:62 - receiveRecordByIdNegative:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 21.48.52 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:66 - receiveRecordById[0] Record with id = 1 not found
03.03 21.48.52 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:67 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:39 - loadListPositive:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:26 - initRecord[0]: Start initiate record
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 270
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:33 - Before run:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 21.48.52  INFO	ru.sfedu.crm.lab4.api.UserListTest:47 - loadListNegative:
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 21.48.52 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.01.29  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:98 - addRecordPositive:
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:99 - UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:22 - initRecord[0]: Start initiate record
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 271
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:78 - updateRecordPositive:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:22 - initRecord[0]: Start initiate record
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 272
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:94 - updateRecord[0]: try to merge(update) entity:UserList{id=272, name='UPDATED NAME', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:96 - updateRecord[0]: entity updated!
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:87 - updateRecordNegative:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:22 - initRecord[0]: Start initiate record
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 273
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 22.01.30 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:66 - receiveRecordById[0] Record with id = 200 not found
03.03 22.01.30 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:67 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:94 - updateRecord[0]: try to merge(update) entity:UserList{id=200, name='null', OrdersId=[], phoneNumber='null'}
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:96 - updateRecord[0]: entity updated!
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 22.01.30 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:66 - receiveRecordById[0] Record with id = 200 not found
03.03 22.01.30 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:67 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:65 - deleteRecordPositive:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:22 - initRecord[0]: Start initiate record
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 275
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.list.UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:79 - deleteRecord[1]: entity successfully deleted!
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:72 - deleteRecordNegative:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.list.UserList
03.03 22.01.30 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:84 - deleteRecord[0] Record with id = 1 not found
03.03 22.01.30 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:85 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:50 - receiveRecordByIdPositive:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:22 - initRecord[0]: Start initiate record
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 276
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:58 - receiveRecordByIdNegative:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 22.01.30 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:66 - receiveRecordById[0] Record with id = 1 not found
03.03 22.01.30 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:67 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:35 - loadListPositive:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:22 - initRecord[0]: Start initiate record
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 277
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.01.30  INFO	ru.sfedu.crm.lab4.api.UserListTest:43 - loadListNegative:
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 22.01.30 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.02.23  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:98 - addRecordPositive:
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:99 - UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:22 - initRecord[0]: Start initiate record
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 278
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:78 - updateRecordPositive:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:22 - initRecord[0]: Start initiate record
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 279
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:94 - updateRecord[0]: try to merge(update) entity:UserList{id=279, name='UPDATED NAME', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:96 - updateRecord[0]: entity updated!
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:87 - updateRecordNegative:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:22 - initRecord[0]: Start initiate record
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 280
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 22.02.25 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:66 - receiveRecordById[0] Record with id = 200 not found
03.03 22.02.25 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:67 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:94 - updateRecord[0]: try to merge(update) entity:UserList{id=200, name='null', OrdersId=[], phoneNumber='null'}
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:96 - updateRecord[0]: entity updated!
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 22.02.25 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:66 - receiveRecordById[0] Record with id = 200 not found
03.03 22.02.25 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:67 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:65 - deleteRecordPositive:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:22 - initRecord[0]: Start initiate record
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 282
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.list.UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:79 - deleteRecord[1]: entity successfully deleted!
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:72 - deleteRecordNegative:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.list.UserList
03.03 22.02.25 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:84 - deleteRecord[0] Record with id = 1 not found
03.03 22.02.25 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:85 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:50 - receiveRecordByIdPositive:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:22 - initRecord[0]: Start initiate record
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 283
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:58 - receiveRecordByIdNegative:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.list.UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:62 - receiveRecordById[0]: Receiving complete
03.03 22.02.25 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:66 - receiveRecordById[0] Record with id = 1 not found
03.03 22.02.25 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:67 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:35 - loadListPositive:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.UserListTest:22 - initRecord[0]: Start initiate record
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:111 - addRecord[0]: try to save entity:UserList{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:114 - addRecord[1]: entity saved!
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:115 - addRecord[1]: record id = 284
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:29 - Before run:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:51 - clearTable[0]: clearing table UserList success!
03.03 22.02.25  INFO	ru.sfedu.crm.lab4.api.UserListTest:43 - loadListNegative:
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserList
03.03 22.02.25 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.09.34  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.09.35 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:46 - clearTable[0]: try to clear table UserSet
03.03 22.11.01  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.11.02 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.11.02 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.11.02  INFO	ru.sfedu.crm.lab4.api.UserSetTest:33 - loadListPositive:
03.03 22.11.02 DEBUG	ru.sfedu.crm.lab4.api.UserSetTest:20 - initRecord[0]: Start initiate record
03.03 22.11.02 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:125 - addRecord[0]: try to save entity:UserSet{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.11.02 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:128 - addRecord[1]: entity saved!
03.03 22.11.02 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:129 - addRecord[1]: record id = 285
03.03 22.11.02 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserSet
03.03 22.11.02 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.11.07  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.11.09 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.11.09 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.11.09  INFO	ru.sfedu.crm.lab4.api.UserSetTest:96 - addRecordPositive:
03.03 22.11.09  INFO	ru.sfedu.crm.lab4.api.UserSetTest:97 - UserSet
03.03 22.11.09 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserSet
03.03 22.11.09 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.11.09 DEBUG	ru.sfedu.crm.lab4.api.UserSetTest:20 - initRecord[0]: Start initiate record
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:125 - addRecord[0]: try to save entity:UserSet{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:128 - addRecord[1]: entity saved!
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:129 - addRecord[1]: record id = 286
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:76 - updateRecordPositive:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.UserSetTest:20 - initRecord[0]: Start initiate record
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:125 - addRecord[0]: try to save entity:UserSet{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:128 - addRecord[1]: entity saved!
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:129 - addRecord[1]: record id = 287
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:108 - updateRecord[0]: try to merge(update) entity:UserSet{id=287, name='UPDATED NAME', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:110 - updateRecord[0]: entity updated!
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:85 - updateRecordNegative:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.UserSetTest:20 - initRecord[0]: Start initiate record
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:125 - addRecord[0]: try to save entity:UserSet{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:128 - addRecord[1]: entity saved!
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:129 - addRecord[1]: record id = 288
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:74 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.set.UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - receiveRecordById[0]: Receiving complete
03.03 22.11.10 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:80 - receiveRecordById[0] Record with id = 200 not found
03.03 22.11.10 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:81 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:108 - updateRecord[0]: try to merge(update) entity:UserSet{id=200, name='null', OrdersId=[], phoneNumber='null'}
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:110 - updateRecord[0]: entity updated!
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:74 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.set.UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - receiveRecordById[0]: Receiving complete
03.03 22.11.10 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:80 - receiveRecordById[0] Record with id = 200 not found
03.03 22.11.10 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:81 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:63 - deleteRecordPositive:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.UserSetTest:20 - initRecord[0]: Start initiate record
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:125 - addRecord[0]: try to save entity:UserSet{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:128 - addRecord[1]: entity saved!
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:129 - addRecord[1]: record id = 290
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:90 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.set.UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:93 - deleteRecord[1]: entity successfully deleted!
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:70 - deleteRecordNegative:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:90 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.set.UserSet
03.03 22.11.10 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:98 - deleteRecord[0] Record with id = 1 not found
03.03 22.11.10 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:99 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:48 - receiveRecordByIdPositive:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.UserSetTest:20 - initRecord[0]: Start initiate record
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:125 - addRecord[0]: try to save entity:UserSet{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:128 - addRecord[1]: entity saved!
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:129 - addRecord[1]: record id = 291
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:74 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.set.UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - receiveRecordById[0]: Receiving complete
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:56 - receiveRecordByIdNegative:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:74 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.set.UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - receiveRecordById[0]: Receiving complete
03.03 22.11.10 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:80 - receiveRecordById[0] Record with id = 1 not found
03.03 22.11.10 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:81 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:33 - loadListPositive:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.UserSetTest:20 - initRecord[0]: Start initiate record
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:125 - addRecord[0]: try to save entity:UserSet{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:128 - addRecord[1]: entity saved!
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:129 - addRecord[1]: record id = 292
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.11.10  INFO	ru.sfedu.crm.lab4.api.UserSetTest:41 - loadListNegative:
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserSet
03.03 22.11.10 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.17.26  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.17.27 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:96 - addRecordPositive:
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:97 - UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.UserSetTest:20 - initRecord[0]: Start initiate record
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:125 - addRecord[0]: try to save entity:UserSet{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:128 - addRecord[1]: entity saved!
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:129 - addRecord[1]: record id = 293
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:76 - updateRecordPositive:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.UserSetTest:20 - initRecord[0]: Start initiate record
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:125 - addRecord[0]: try to save entity:UserSet{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:128 - addRecord[1]: entity saved!
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:129 - addRecord[1]: record id = 294
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:108 - updateRecord[0]: try to merge(update) entity:UserSet{id=294, name='UPDATED NAME', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:110 - updateRecord[0]: entity updated!
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:85 - updateRecordNegative:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.UserSetTest:20 - initRecord[0]: Start initiate record
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:125 - addRecord[0]: try to save entity:UserSet{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:128 - addRecord[1]: entity saved!
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:129 - addRecord[1]: record id = 295
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:74 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.set.UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - receiveRecordById[0]: Receiving complete
03.03 22.17.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:80 - receiveRecordById[0] Record with id = 200 not found
03.03 22.17.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:81 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:108 - updateRecord[0]: try to merge(update) entity:UserSet{id=200, name='null', OrdersId=[], phoneNumber='null'}
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:110 - updateRecord[0]: entity updated!
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:74 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.set.UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - receiveRecordById[0]: Receiving complete
03.03 22.17.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:80 - receiveRecordById[0] Record with id = 200 not found
03.03 22.17.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:81 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:63 - deleteRecordPositive:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.UserSetTest:20 - initRecord[0]: Start initiate record
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:125 - addRecord[0]: try to save entity:UserSet{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:128 - addRecord[1]: entity saved!
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:129 - addRecord[1]: record id = 297
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:90 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.set.UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:93 - deleteRecord[1]: entity successfully deleted!
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:70 - deleteRecordNegative:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:90 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.set.UserSet
03.03 22.17.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:98 - deleteRecord[0] Record with id = 1 not found
03.03 22.17.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:99 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:48 - receiveRecordByIdPositive:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.UserSetTest:20 - initRecord[0]: Start initiate record
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:125 - addRecord[0]: try to save entity:UserSet{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:128 - addRecord[1]: entity saved!
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:129 - addRecord[1]: record id = 298
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:74 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.set.UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - receiveRecordById[0]: Receiving complete
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:56 - receiveRecordByIdNegative:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:74 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.set.UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:76 - receiveRecordById[0]: Receiving complete
03.03 22.17.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:80 - receiveRecordById[0] Record with id = 1 not found
03.03 22.17.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:81 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:33 - loadListPositive:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.UserSetTest:20 - initRecord[0]: Start initiate record
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:125 - addRecord[0]: try to save entity:UserSet{id=0, name='John Smith', OrdersId=[1, 2, 3], phoneNumber='+78005553535'}
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:128 - addRecord[1]: entity saved!
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:129 - addRecord[1]: record id = 299
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:27 - Before run:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserSet success!
03.03 22.17.28  INFO	ru.sfedu.crm.lab4.api.UserSetTest:41 - loadListNegative:
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserSet
03.03 22.17.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.26.31  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.26.31  INFO	ru.sfedu.crm.lab4.api.UserMapTest:35 - loadListPositive:
03.03 22.26.31 DEBUG	ru.sfedu.crm.lab4.api.UserMapTest:22 - initRecord[0]: Start initiate record
03.03 22.26.33 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:139 - addRecord[0]: try to save entity:UserMap{id=0, name='John Smith', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.26.33 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:142 - addRecord[1]: entity saved!
03.03 22.26.33 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:143 - addRecord[1]: record id = 300
03.03 22.26.33 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.26.33 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.29.19  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.29.22 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.29.22 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.29.22  INFO	ru.sfedu.crm.lab4.api.UserMapTest:35 - loadListPositive:
03.03 22.29.22 DEBUG	ru.sfedu.crm.lab4.api.UserMapTest:22 - initRecord[0]: Start initiate record
03.03 22.29.22 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:139 - addRecord[0]: try to save entity:UserMap{id=0, name='John Smith', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.29.22 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:142 - addRecord[1]: entity saved!
03.03 22.29.22 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:143 - addRecord[1]: record id = 301
03.03 22.29.22 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.29.22 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.29.26  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.29.27 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:98 - addRecordPositive:
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:99 - UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.UserMapTest:22 - initRecord[0]: Start initiate record
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:139 - addRecord[0]: try to save entity:UserMap{id=0, name='John Smith', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:142 - addRecord[1]: entity saved!
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:143 - addRecord[1]: record id = 302
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:78 - updateRecordPositive:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.UserMapTest:22 - initRecord[0]: Start initiate record
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:139 - addRecord[0]: try to save entity:UserMap{id=0, name='John Smith', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:142 - addRecord[1]: entity saved!
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:143 - addRecord[1]: record id = 303
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:122 - updateRecord[0]: try to merge(update) entity:UserMap{id=303, name='UPDATED NAME', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:124 - updateRecord[0]: entity updated!
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:87 - updateRecordNegative:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.UserMapTest:22 - initRecord[0]: Start initiate record
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:139 - addRecord[0]: try to save entity:UserMap{id=0, name='John Smith', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:142 - addRecord[1]: entity saved!
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:143 - addRecord[1]: record id = 304
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.map.UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:90 - receiveRecordById[0]: Receiving complete
03.03 22.29.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:94 - receiveRecordById[0] Record with id = 200 not found
03.03 22.29.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:95 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:122 - updateRecord[0]: try to merge(update) entity:UserMap{id=200, name='null', OrdersId={}, phoneNumber='null'}
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:124 - updateRecord[0]: entity updated!
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.map.UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:90 - receiveRecordById[0]: Receiving complete
03.03 22.29.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:94 - receiveRecordById[0] Record with id = 200 not found
03.03 22.29.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:95 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:65 - deleteRecordPositive:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.UserMapTest:22 - initRecord[0]: Start initiate record
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:139 - addRecord[0]: try to save entity:UserMap{id=0, name='John Smith', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:142 - addRecord[1]: entity saved!
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:143 - addRecord[1]: record id = 306
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:104 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.map.UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:107 - deleteRecord[1]: entity successfully deleted!
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:72 - deleteRecordNegative:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:104 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.map.UserMap
03.03 22.29.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - deleteRecord[0] Record with id = 1 not found
03.03 22.29.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:113 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:50 - receiveRecordByIdPositive:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.UserMapTest:22 - initRecord[0]: Start initiate record
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:139 - addRecord[0]: try to save entity:UserMap{id=0, name='John Smith', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:142 - addRecord[1]: entity saved!
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:143 - addRecord[1]: record id = 307
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.map.UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:90 - receiveRecordById[0]: Receiving complete
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:58 - receiveRecordByIdNegative:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.map.UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:90 - receiveRecordById[0]: Receiving complete
03.03 22.29.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:94 - receiveRecordById[0] Record with id = 1 not found
03.03 22.29.28 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:95 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:35 - loadListPositive:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.UserMapTest:22 - initRecord[0]: Start initiate record
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:139 - addRecord[0]: try to save entity:UserMap{id=0, name='John Smith', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:142 - addRecord[1]: entity saved!
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:143 - addRecord[1]: record id = 308
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.29.28  INFO	ru.sfedu.crm.lab4.api.UserMapTest:43 - loadListNegative:
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.29.28 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.38.05  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:98 - addRecordPositive:
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:99 - UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.UserMapTest:22 - initRecord[0]: Start initiate record
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:139 - addRecord[0]: try to save entity:UserMap{id=0, name='John Smith', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:142 - addRecord[1]: entity saved!
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:143 - addRecord[1]: record id = 309
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:78 - updateRecordPositive:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.UserMapTest:22 - initRecord[0]: Start initiate record
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:139 - addRecord[0]: try to save entity:UserMap{id=0, name='John Smith', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:142 - addRecord[1]: entity saved!
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:143 - addRecord[1]: record id = 310
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:122 - updateRecord[0]: try to merge(update) entity:UserMap{id=310, name='UPDATED NAME', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:124 - updateRecord[0]: entity updated!
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:87 - updateRecordNegative:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.UserMapTest:22 - initRecord[0]: Start initiate record
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:139 - addRecord[0]: try to save entity:UserMap{id=0, name='John Smith', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:142 - addRecord[1]: entity saved!
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:143 - addRecord[1]: record id = 311
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.map.UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:90 - receiveRecordById[0]: Receiving complete
03.03 22.38.07 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:94 - receiveRecordById[0] Record with id = 200 not found
03.03 22.38.07 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:95 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:122 - updateRecord[0]: try to merge(update) entity:UserMap{id=200, name='null', OrdersId={}, phoneNumber='null'}
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:124 - updateRecord[0]: entity updated!
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.map.UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:90 - receiveRecordById[0]: Receiving complete
03.03 22.38.07 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:94 - receiveRecordById[0] Record with id = 200 not found
03.03 22.38.07 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:95 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:65 - deleteRecordPositive:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.UserMapTest:22 - initRecord[0]: Start initiate record
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:139 - addRecord[0]: try to save entity:UserMap{id=0, name='John Smith', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:142 - addRecord[1]: entity saved!
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:143 - addRecord[1]: record id = 313
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:104 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.map.UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:107 - deleteRecord[1]: entity successfully deleted!
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:72 - deleteRecordNegative:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:104 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.map.UserMap
03.03 22.38.07 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:112 - deleteRecord[0] Record with id = 1 not found
03.03 22.38.07 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:113 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:50 - receiveRecordByIdPositive:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.UserMapTest:22 - initRecord[0]: Start initiate record
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:139 - addRecord[0]: try to save entity:UserMap{id=0, name='John Smith', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:142 - addRecord[1]: entity saved!
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:143 - addRecord[1]: record id = 314
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.map.UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:90 - receiveRecordById[0]: Receiving complete
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:58 - receiveRecordByIdNegative:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.map.UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:90 - receiveRecordById[0]: Receiving complete
03.03 22.38.07 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:94 - receiveRecordById[0] Record with id = 1 not found
03.03 22.38.07 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:95 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:35 - loadListPositive:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.UserMapTest:22 - initRecord[0]: Start initiate record
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:139 - addRecord[0]: try to save entity:UserMap{id=0, name='John Smith', OrdersId={1=CLASSIC, 2=CLASSIC, 3=AMBIENT}, phoneNumber='+78005553535'}
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:142 - addRecord[1]: entity saved!
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:143 - addRecord[1]: record id = 315
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:29 - Before run:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:60 - clearTable[0]: try to clear table UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:65 - clearTable[0]: clearing table UserMap success!
03.03 22.38.07  INFO	ru.sfedu.crm.lab4.api.UserMapTest:43 - loadListNegative:
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserMap
03.03 22.38.07 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 23.02.48  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:29 - Before run:
03.03 23.02.48  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:35 - loadListPositive:
03.03 23.02.48 DEBUG	ru.sfedu.crm.lab4.api.UserCollectionTest:22 - initRecord[0]: Start initiate record
03.03 23.02.49 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:153 - addRecord[0]: try to save entity:UserCollection{id=0, name='John Smith', order=[OrdersId{orderType=CLASSIC, id=2}, OrdersId{orderType=CLASSIC, id=2}], phoneNumber='+78005553535'}
03.03 23.02.49 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:161 - addRecord[0]: saving error!
03.03 23.02.49 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:162 - addRecord[1]:javax.persistence.PersistenceException: org.hibernate.exception.ConstraintViolationException: could not execute statement
03.03 23.02.49 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserCollection
03.03 23.02.49 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 23.08.20  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:29 - Before run:
03.03 23.08.21 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - clearTable[0]: try to clear table UserCollection
03.03 23.08.21 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:93 - clearTable[0]: clearing table UserCollection success!
03.03 23.08.21  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:35 - loadListPositive:
03.03 23.08.21 DEBUG	ru.sfedu.crm.lab4.api.UserCollectionTest:22 - initRecord[0]: Start initiate record
03.03 23.08.21 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:153 - addRecord[0]: try to save entity:UserCollection{id=0, name='John Smith', order=[Order{orderType=CLASSIC, id=1}], phoneNumber='+78005553535'}
03.03 23.08.22 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:156 - addRecord[1]: entity saved!
03.03 23.08.22 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:157 - addRecord[1]: record id = 317
03.03 23.08.22 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserCollection
03.03 23.08.22 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 23.08.40  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:29 - Before run:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - clearTable[0]: try to clear table UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:93 - clearTable[0]: clearing table UserCollection success!
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:98 - addRecordPositive:
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:99 - UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.UserCollectionTest:22 - initRecord[0]: Start initiate record
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:153 - addRecord[0]: try to save entity:UserCollection{id=0, name='John Smith', order=[Order{orderType=CLASSIC, id=1}], phoneNumber='+78005553535'}
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:156 - addRecord[1]: entity saved!
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:157 - addRecord[1]: record id = 318
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:29 - Before run:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - clearTable[0]: try to clear table UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:93 - clearTable[0]: clearing table UserCollection success!
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:78 - updateRecordPositive:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.UserCollectionTest:22 - initRecord[0]: Start initiate record
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:153 - addRecord[0]: try to save entity:UserCollection{id=0, name='John Smith', order=[Order{orderType=CLASSIC, id=1}], phoneNumber='+78005553535'}
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:156 - addRecord[1]: entity saved!
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:157 - addRecord[1]: record id = 319
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:136 - updateRecord[0]: try to merge(update) entity:UserCollection{id=319, name='UPDATED NAME', order=[Order{orderType=CLASSIC, id=1}], phoneNumber='+78005553535'}
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:138 - updateRecord[0]: entity updated!
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:29 - Before run:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - clearTable[0]: try to clear table UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:93 - clearTable[0]: clearing table UserCollection success!
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:87 - updateRecordNegative:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.UserCollectionTest:22 - initRecord[0]: Start initiate record
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:153 - addRecord[0]: try to save entity:UserCollection{id=0, name='John Smith', order=[Order{orderType=CLASSIC, id=1}], phoneNumber='+78005553535'}
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:156 - addRecord[1]: entity saved!
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:157 - addRecord[1]: record id = 320
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:102 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.collection.UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:104 - receiveRecordById[0]: Receiving complete
03.03 23.08.41 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:108 - receiveRecordById[0] Record with id = 200 not found
03.03 23.08.41 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:109 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:136 - updateRecord[0]: try to merge(update) entity:UserCollection{id=200, name='null', order=[], phoneNumber='null'}
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:138 - updateRecord[0]: entity updated!
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:102 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.collection.UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:104 - receiveRecordById[0]: Receiving complete
03.03 23.08.41 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:108 - receiveRecordById[0] Record with id = 200 not found
03.03 23.08.41 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:109 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:29 - Before run:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - clearTable[0]: try to clear table UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:93 - clearTable[0]: clearing table UserCollection success!
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:65 - deleteRecordPositive:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.UserCollectionTest:22 - initRecord[0]: Start initiate record
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:153 - addRecord[0]: try to save entity:UserCollection{id=0, name='John Smith', order=[Order{orderType=CLASSIC, id=1}], phoneNumber='+78005553535'}
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:156 - addRecord[1]: entity saved!
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:157 - addRecord[1]: record id = 322
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:118 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.collection.UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:121 - deleteRecord[1]: entity successfully deleted!
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:29 - Before run:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - clearTable[0]: try to clear table UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:93 - clearTable[0]: clearing table UserCollection success!
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:72 - deleteRecordNegative:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:118 - deleteRecord[0]: try to delete entity:class ru.sfedu.crm.lab4.model.collection.UserCollection
03.03 23.08.41 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:126 - deleteRecord[0] Record with id = 1 not found
03.03 23.08.41 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:127 - deleteRecord[1]:java.lang.IllegalArgumentException: attempt to create delete event with null entity
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:29 - Before run:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - clearTable[0]: try to clear table UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:93 - clearTable[0]: clearing table UserCollection success!
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:50 - receiveRecordByIdPositive:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.UserCollectionTest:22 - initRecord[0]: Start initiate record
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:153 - addRecord[0]: try to save entity:UserCollection{id=0, name='John Smith', order=[Order{orderType=CLASSIC, id=1}], phoneNumber='+78005553535'}
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:156 - addRecord[1]: entity saved!
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:157 - addRecord[1]: record id = 323
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:102 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.collection.UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:104 - receiveRecordById[0]: Receiving complete
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:29 - Before run:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - clearTable[0]: try to clear table UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:93 - clearTable[0]: clearing table UserCollection success!
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:58 - receiveRecordByIdNegative:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:102 - receiveRecordById[0]: try to receive entity by id:class ru.sfedu.crm.lab4.model.collection.UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:104 - receiveRecordById[0]: Receiving complete
03.03 23.08.41 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:108 - receiveRecordById[0] Record with id = 1 not found
03.03 23.08.41 ERROR	ru.sfedu.crm.lab4.api.DataProviderLab4:109 - receiveRecordById[1]:java.lang.NullPointerException: null
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:29 - Before run:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - clearTable[0]: try to clear table UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:93 - clearTable[0]: clearing table UserCollection success!
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:35 - loadListPositive:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.UserCollectionTest:22 - initRecord[0]: Start initiate record
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:153 - addRecord[0]: try to save entity:UserCollection{id=0, name='John Smith', order=[Order{orderType=CLASSIC, id=1}], phoneNumber='+78005553535'}
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:156 - addRecord[1]: entity saved!
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:157 - addRecord[1]: record id = 324
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:29 - Before run:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:88 - clearTable[0]: try to clear table UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:93 - clearTable[0]: clearing table UserCollection success!
03.03 23.08.41  INFO	ru.sfedu.crm.lab4.api.UserCollectionTest:43 - loadListNegative:
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:28 - loadList[0]: try to load list of entities from table :UserCollection
03.03 23.08.41 DEBUG	ru.sfedu.crm.lab4.api.DataProviderLab4:31 - loadList[0]: Loading success!
